app/src/main/res/values-it/strings.xml

```diff
@@ -0,0 +1,224 @@
+    <string name="amoled_theme_setting">Tema nero</string>
+    <string name="amoled_theme_desc">Abilita il tema nero AMOLED</string>
+    <string name="theme_dialog_apply_button">Applica</string>
+    <string name="material_you_setting" translatable="false">Material You</string>
+    <string name="material_you_setting_desc">Cambia il tema in base al tuo sfondo (Android 12+)</string>
+    <string name="material_you_error">Questa funzionalità è solo per i dispositivi con Android 12 o in su.</string>
+    <string name="goal_card_setting">Stile scheda obbiettivo</string>
+    <string name="goal_card_option1">Classica</string>
+    <string name="goal_card_option2">Compatta</string>
+    <string name="locales_setting_title">Lingua</string>
+    <string name="app_locale_setting">Lingua predefinita</string>
+    <string name="app_locale_setting_desc">Seleziona la lingua preferita</string>
+    <string name="date_format_setting">Formato data</string>
+    <string name="date_format_dialog_title">Cambia formato data</string>
+    <string name="preferred_currency_setting">Valuta preferita</string>
+    <string name="currency_dialog_title">Seleziona la valuta preferita</string>
+    <string name="security_settings_title">Sicurezza</string>
+    <string name="app_lock_setting">Blocco biometrico</string>
+    <string name="app_lock_setting_desc">Usa  la tua impronta digitale o la password di blocco schermo per sbloccare GreenStash</string>
+    <string name="misc_setting_title">Miscellanee</string>
+    <string name="license_setting">Licenza &amp; Riconoscimenti</string>
+    <string name="license_setting_desc">Mostra le informazioni sulla licenza open source.</string>
+    <string name="app_info_setting">Informazioni App</string>
+    <string name="app_info_setting_desc">Mostra la versione dell\'app e altri link utili.</string>
+
+    <!-- Goal Card Style -->
+    <string name="goal_card_settings_header">Stile Obbiettivo</string>
+    <string name="goal_card_settings_tip">Suggerimento! Puoi scorrere una scheda in stile compatto a sinistra o destra per modificare o eliminare un obbiettivo di risparmio.</string>
+
+    <!-- OSL Screen -->
+    <string name="osl_screen_header">Librerie Open Source</string>
+
+    <!-- About Screen -->
+    <string name="about_screen_header">Informazioni</string>
+    <string name="about_readme_title">README</string>
+    <string name="about_readme_desc">Vai a vedere il repository di GitHub e il README dell\'app.</string>
+    <string name="about_privacy_title">Informativa sulla Privacy</string>
+    <string name="about_privacy_desc">Leggi l\'informativa sulla privacy e i termini di servizio su GitHub.</string>
+    <string name="about_gh_issue_title">Issues di GitHub</string>
+    <string name="about_gh_issue_desc">Invia segnalazioni di bug or richieste di funzionalità creando una issue su GitHub.</string>
+    <string name="about_telegram_title">Gruppo Telegram/string>
```

Comment:
Missing closing tag here, Should be: ` <string name="about_telegram_title">Gruppo Telegram</string>`

Reply:
WHAT????!? THIS WAS THE ERROR???!? oh my god i'm speechless